Face-Detection.py

```
@@ -336,7 +335,7 @@ def detect_faces(f_cascade, colored_img, scaleFactor=1.1):
 plt.imshow(convertToRGB(faces_detected_img))
 
 # ## Haar and LBP Results Analysis
-
+sqdwefreg
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)